fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Reorg.java

```diff
@@ -1,14 +0,0 @@
-/*
- * (C) Copyright IBM Corp. 2019
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package com.ibm.fhir.database.utils.model;
-
-/**
- * Representing the SQL function
- */
-public class Reorg {
```

Comment:
I actually agree with Reorg.  
I don't think it should be Reorg Table
You can reorg indices as well and other database objects.

Reply:
well the javadoc and the arguments all say table... how about `Db2Reorg`?
does Reorg apply to other dbs besides Db2?

Reply:
That's a good question.  MSSQL uses REORGANIZE https://docs.microsoft.com/en-us/sql/relational-databases/indexes/reorganize-and-rebuild-indexes?view=sql-server-ver15

Reply:
I'm OK resolving.